samples/default-authorizationserver/samples-default-authorizationserver.gradle

```diff
@@ -8,6 +8,10 @@ group = project.rootProject.group
 version = project.rootProject.version
 sourceCompatibility = "17"
 
+test {
+	useJUnitPlatform()
```

Comment:
This can be removed since it's automatically applied by the `SpringJavaPlugin`

Reply:
I spent quite a while figuring out why the tests don't run locally and in CI, in my opinion this config should have been added in the "Upgrade to JUnit 5" commit.

Did you try it out and are tests running (and failing the build) on your machine without explicitly enabling JUnit 5?

Reply:
I just realized that the samples do not use the `io.spring.convention.spring-module` plugin, so `SpringJavaPlugin` is not applied as I previously mentioned. Sorry for the confusion there.

However, I'm wondering why the tests are failing on your end? The tests in `default-authorizationserver` sample are passing locally and in CI for quite some time now.

Reply:
Sorry, I didn't make myself fully clear. The tests are not failing in their current state.

What I meant was: if Gradle says the build was successful, that does not necessarily mean that it ran tests. To check if it does, I like to make a test fail, e.g. by throwing a RuntimeException. The failing test should also lead to a failing build.

For CI, I checked the logs of several commits for the test output – I think there's a DB-related message. This message did not appear after the upgrade to JUnit 5.